_editorial_policies/02_editor_instructions.md

```diff
@@ -42,9 +46,9 @@ Once an editor has handled the pre-review steps described above, the review proc
 - Begins the process of conveying reviews to authors once two have been received, unless the need for additional expertise requires a third review. In addition to the two required peer reviews, a student review must also be obtained. A student reivew may be edited to ensure anonymity and proper tone, but not for substance.
 - Handles any potential conflict of interests disclosed by reviewers consistent with [editorial policy](https://livecomsjournal.github.io/policies/editorial_board/)
 - Makes reviewers aware of the [review criteria](https://livecomsjournal.github.io/authors/policies/reviewer_information), including category-specific review criteria
-- Performs a check that the GitHub repository is in order, or ensure the reviewers do so.
+- Performs a check that the GitHub repository contains all of the files in the submission.
 - Ensures reviews are submitted and analyzed in a timely manner (all reviews should be submitted in no more than about 25 days), reaching out to remind reviewers as needed and solicit additional reviews if reviewers are too slow or their analysis conflicts
 - Potentially helps ensure reviewer feedback is fair
 - Helps preserve anonymity of the reviewers, especially the student reviewer, by editing the reviews if necessary.  Note that the reviewers may choose to waive anonymity by specifically referring to GitHub issues they have filed or otherwise explicitly making it clear they intend to identify themselves.
-- Makes a recommendation regarding acceptance, acceptance with minor or major revision, or rejection (keeping in mind the content of the presubmission letter), and communicates this recommendation to the Lead Editor who makes a final decision. Normally the Lead Editor will follow the handling editor's recommendations, but this process is helpful since the Lead will have approved the presubmission letter and thus needs to be involved if a less than favorable decision is to be made.
-- If the decision is that the authors must revise, provides appropriate direction (with the Lead Editor) about which comments to address if needed
+- Decides regarding acceptance, acceptance with minor or major revision, or rejection (keeping in mind the content of the presubmission letter), and communicates this recommendation to the Lead Editor who makes a final decision. Normally the Section Lead Editor will not be involved, but if they notified the Associate Editor that they should be involved during the process, they should approve any decision. 
```

Comment:
I would replace the final "they" with "Section Lead Editor" for clarity

Reply:
Change made.